app/src/main/java/com/example/footprnt/Map/MapFragment.java

```diff
@@ -937,4 +964,19 @@ private void toggleMenuItem(MenuItem menuItem, int id) {
             }
         }
     }
+
+    @Override
+    public void onConnected(@Nullable Bundle bundle) {
+
```

Comment:
I'd encourage you to put "// TODO"s or "// Do nothing"s in here to make it clear whether this method is a work in progress or empty.